sections/installation.js

```diff
@@ -390,4 +390,36 @@ export default [
       });
     `
   },
+  {
+    type: "heading",
+    size: "md",
+    content: "log",
+    href: "Installation-log"
+  },
+  {
+    type: "text",
+    content: [
+      "Knex contains some internal log functions for printing warnings, errors, deprecations, and debug information when applicable.",
+      "These log functions typically log to the console, but can be overwritten using the log option and providing alternative functions.",
+      "Can be different log functions for seperate knex instances.",
```

Comment:
Also it probably sounds better if reworded to be "Different log functions can be used for separate knex instances"